dashboard/app/views/devise/registrations/_old_sign_up_form.html.haml

```diff
@@ -152,9 +152,13 @@
       -# Temporarily use student_terms so that existing translations get used.
       != t('signup_form.student_terms')
 
-  .span4
-    .devise_links
-      = render "devise/shared/old_links"
+  .span5{style: "display: flex;"}
+    %div.vertical-or
+      %hr
+      = t("or").upcase
```

Comment:
is there a difference between using single or double quotes here?

Reply:
nope. i believe single-vs-double quotes only matters in Ruby when you have string interpolation